.sync.yml

```diff
@@ -1,24 +0,0 @@
----
-.travis.yml:
-  user: camptocamp
-  secure: "XgiXVp8K+UjVAQFVC4qBt6BPNBMhNGBHgpKlB5f4bpiY2wIZjlLNLdhilBnPqZcuJAmauTcdhmI0C7Iys4aKPsS/oQtZAjRjPvgXtvA2iza+aiPUrrjQASDOWlfuGRj1yMqNXDDDC18RZO+ajoruLO32Xvi2nf9exxm64BrSehw="
-  branches:
-    - master
-    - /^\d/
-  deploy_to_forge:
-    enabled: true
-    tag_regex: "^\\d"
-Gemfile:
-  optional:
-    ':development, :unit_tests':
-      - gem: inifile
-      - gem: webmock
-    ':development':
-      - gem: 'github_changelog_generator'
-
-Rakefile:
-  changelog_version_tag_pattern: '%s'
```

Comment:
This may be something we do want to consider keeping. Rather than prefixing with `v` everywhere.

Reply:
I've never really seen the point of the `v` prefix, which is why we went with bare tags in our projects.

Reply:
I agree with that. However, I prefer consistency more so I won't argue for changing it in a project. In some new projects I also started without a `v` prefix.